Gemfile

```diff
@@ -154,6 +151,9 @@ gem 'rack-cors'
 
 group :production, :qa, :sandbox, :staging do
```

Comment:
@inulty-dfe I have a gut feeling that production_aks, qa_aks, sandbox_aks and staging_aks will also need adding here

Reply:
This is interesting. These environments are only related to the deployment, no? I don't think the application should be aware of them. There shouldn't be any difference between `env` and `env_aks` from a bundling point of view.

Reply:
We set the different envs in `config/environments` and settings in `config/settings` and some app config from `terraform/aks/workspace_variables/app_config.yml` so if any of those mean that what that's reading might be different then may be worth a look. I'm not overly familiar with Ruby/Rails/Rack etc.

Reply:
This doesn't feel right from the application perspective. It's bleeding deployment configuration into application code. Functionally there are 3 environments in the Rails application. But the application has to be aware of 7 environment names. There is a `DEPLOY_ENV` in the Makefile. Why do we need to mention `_aks` in the `RAILS_ENV` and `RACK_ENV` environment variables in terraform?